addons/api/mirage/generated/factories/auth-method.js

```diff
@@ -11,4 +11,22 @@ export default Factory.extend({
   created_time: () => date.recent(),
   updated_time: () => date.recent(),
   disabled: () => datatype.boolean(),
+
+  attributes: function () {
+    let attrs;
+    switch (this.type) {
```

Comment:
I do asume we will have more auth methods in the future?

Reply:
We already have two types:  `password` and `oidc`.  However, only `oidc` uses the `attributes` field right now.

Reply:
Yep. I was asking about it for the usage of `switch`. I understand we are using a `switch` statement because we expect 3 or more cases in the near future.